python/createCluster.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+# Generic/Built-in
+import json
+import argparse
+
+# Other Libs
+import texttable as tt
+
+# Owned
+from cbcapi.cbc_api import cbc_api_get, cbc_api_post
+
+__author__ = 'Jonathan Giffard'
+__copyright__ = 'Copyright 2021, Couchbase'
+__credits__ = ['Jonathan Giffard']
+__license__ = 'MIT License'
```

Comment:
is it intentional to have different licenses per different files? One has MIT another GPL